tests/translator/test_translator.py

```diff
@@ -657,6 +660,93 @@ def _do_transform(self, document, parameter_values=get_template_parameter_values
         return output_fragment
 
 
+class TestApiAlwaysDeploy(TestCase):
```

Comment:
can we have some transform tests with `time` mocked so we know it kind of work end to end

Reply:
It is there.